### PR TITLE
[CLOUDTRUST-1262] Get groups

### PR DESCRIPTION
**Pull request by bsoniam**
Added get groups endpoint 
Added authorization for GetRoles: anyone can query and get the roles of his realm

### CODE REVIEW

File: cmd/keycloakb/keycloak_bridge.go [hunk 1]
```diff
@@ -594,6 +597,9 @@ func main() {
 		managementSubroute.Path("/realms/{realm}/clients/{clientID}/roles").Methods("GET").Handler(getClientRolesHandler)
 		managementSubroute.Path("/realms/{realm}/clients/{clientID}/roles").Methods("POST").Handler(createClientRolesHandler)
 
+		//groups
+		managementSubroute.Path("/realms/{realm:[a-zA-Z0-9_-]+}/groups").Methods("GET").Handler(getGroupsHandler)
```
Comment (harture) [inline cmd/keycloakb/keycloak_bridge.go]: the regexp is not needed anymore here. We manage it in http.go

File: configs/authorization.json [hunk 2]
```diff
@@ -119,6 +129,16 @@
         "master": {
           "integrator_agent": {}
         },
+      "GetGroups": {
+        "master": {
```
Comment (harture) [inline configs/authorization.json]: I think DEP should also be added

File: configs/authorization.json [hunk 3]
```diff
@@ -119,6 +129,16 @@
...
+          "*": {}
+        }
+      },
+      "GetRoles": {
+        "master": {
```
Comment (harture) [inline configs/authorization.json]: DEP may be useful

File: configs/authorization.json [hunk 4]
```diff
@@ -245,6 +265,16 @@
           "*": {}
         }
       },
+      "GetGroups": {
+        "master": {
```
Comment (harture) [inline configs/authorization.json]: should be DEP instead of master

File: configs/authorization.json [hunk 5]
```diff
@@ -245,6 +265,16 @@
...
+          "*": {}
+        }
+      },
+      "GetRoles": {
+        "master": {
```
Comment (harture) [inline configs/authorization.json]: should be DEP instead of master

File: configs/authorization.json (hunk 3 above)
Comment (bsoniam) [reply configs/authorization.json]: DONE

File: configs/authorization.json (hunk 4 above)
Comment (bsoniam) [reply configs/authorization.json]: DONE

File: configs/authorization.json (hunk 2 above)
Comment (bsoniam) [reply configs/authorization.json]: DONE

File: cmd/keycloakb/keycloak_bridge.go (hunk 1 above)
Comment (bsoniam) [reply cmd/keycloakb/keycloak_bridge.go]: DONE

**Pull request merged**